src/main/java/com/datalogics/pdf/security/LunaHsmManager.java

```diff
@@ -22,12 +22,17 @@
 
 /**
  * This class allows for connecting to a Luna SA Hsm Device.
+ *
+ * <p>
+ * Note: Once an instance of LunaHsmManager goes into a
+ * {@link com.datalogics.pdf.security.HsmManager.ConnectionState#DISCONNECTED} state during logout from the LunaSA you
+ * will need to create a new instance of this class if you need to login to the LunaSA device again.
```

Comment:
@datalogics-robl 

I debated back and forth with the notion of where to put this note. But each implementation of the  hsmLogin & hsmLogout methods will be different depending on what new HsmManagers get created down the line. 

Let's say that a user/developer implements a new HsmManager for a new HSM device and they don't care about checking the state of the HsmManager when logging in because their device supports multiple logins. In this case putting this note in the HsmManger interface would be useless because they are choosing how to implement their version of the HsmManager.